tools/torchfix/README.md

```diff
@@ -48,3 +48,19 @@ To enable them, use standard flake8 configuration options for the plugin mode or
 
 If you encounter a bug or some other problem with TorchFix, please file an issue on
 https://github.com/pytorch/test-infra/issues, mentioning [TorchFix] in the title.
+
+
+## Rules
+
+### TOR001 Use of removed function
+
+#### torch.solve
```

Comment:
Is there a doc/page that calls out this function as deprecated? If so, what do you think of linking to it here?

Reply:
No, it's removed from the docs.
The YAML in the PR has links to deprecate and remove PRs, I plan later to use these links to generate some docs.